src/main.ts

```diff
@@ -1,4 +1,39 @@
 import pino from 'pino';
+const logger = pino({
+  level: process.env.LOG_LEVEL || 'info',
+});
+
+import apm from 'elastic-apm-node';
+/* istanbul ignore next */
+{
+  const isApmEnabled = !!process.env.ENABLE_APM;
```

Comment:
Could be a bit surprising that `ENABLE_APM=false ...` leads to `!!'false' === true`

Reply:
Changed, thoughts?